index.html

```diff
@@ -164,6 +164,27 @@ <h3 class="skill">Skills</h3>
                 </div>
             </div>
         </section>
+        <footer class="contact">
+            <h2>I'm always <br> interested in hearing about new projects, 
+                so if you'd like to chat please get in</h2>
+            <form class="ip-type" action="https://formspree.io/f/xzbokgee">
+                <input class="Name" type="text" placeholder="    Full Name" maxlength="30" required>
+                <input class="email" type="email" placeholder="   Email address" required>
+                <textarea name="msgt1" class="t1" cols="30" rows="10" maxlength="500" placeholder="  Enter text here"></textarea>
```

Comment:
- [ ] The ```textarea``` doesn't have a required attribute, kindly add ```required``` to this input. it's a requirement